codegen/smithy-aws-ruby-codegen/src/main/java/software/amazon/smithy/aws/ruby/codegen/HttpBearerAuth.java

```diff
@@ -49,12 +49,18 @@ public List<ClientConfig> getAdditionalClientConfig(GenerationContext context) {
                 @see AWS::SDK::Core::CREDENTIALS_PROVIDER_CHAIN
                 """;
 
+        String identityProviderChain = "AWS::SDK::Core::HTTPBearerProviderChain";
+        String defaultIdentity = "Hearth::Identities::HTTPBearer.new(token: 'token')";
+        String defaultConfigValue = "cfg[:stub_responses] ? %s.new(proc { %s }) : nil"
```

Comment:
Naming nit: calling it defaultConfig value is a little confusing here to me since this is only for stubbed responses.

Reply:
Yes

Reply:
It is a default value though.. either nil or a stub response.